rlp/sedes/serializable.py

```diff
@@ -236,6 +236,11 @@ def __len__(self):
     def __eq__(self, other):
         return isinstance(other, Serializable) and hash(self) == hash(other)
 
+    def __getstate__(self):
+        state = self.__dict__.copy()
+        state['_hash_cache'] = None
```

Comment:
It's subtle enough that it seems worth a comment explaining (about `hash()` not being stable across processes).

Reply:
Makes sense